po/fr.po

```diff
@@ -1738,7 +1735,7 @@ msgid "start download on startup"
 msgstr "commencer le téléchargement au démarrage"
 
 #: src/pbview.cpp:64
-#, fuzzy, c-format
+#, c-format
```

Comment:
The translation ends with `%s`, whereas the original message doesn't; are you sure the translation is correct?